conf/cmake/user.cmake

```diff
@@ -38,16 +40,16 @@ set(UserDefLibPaths)
 
 set(UserDefLibs)
 
-set(UserDefOutput)
+set(UserDefOutput "product")
 
 
 
```

Comment:
@Ali-Nasrolahi , there are too many new lines in the file :)
Right now it is OK, but please do not forget at the end to make some cleanups.
Thank you!

Reply:
Sure no problem at all.